styles.css

```diff
@@ -0,0 +1,88 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  height: 718px;
+  width: 375px;
```

Comment:
- [ ] It is not good practice to use fixed height and width to HTML elements for it makes your page not responsive. For example, this is what I am getting due to the fixed pixel unit width and height you have used. You can remove them and check the difference.
![layout-dueto-fixed-width-height](https://user-images.githubusercontent.com/44057863/164801971-af1ede25-2dc7-4333-8a2d-c99d5cda2c89.png)